source/scss/_pf-basic-icons.scss

```diff
@@ -219,3 +219,12 @@
 .#{$pf-icon-prefix}-zone:before {
   content: $pficon-var-zone;
 }
+.pf-icon-circle-lg {
```

Comment:
Oh, yes! I've haven't though it that way. If we decouple them `pf-icon--circle`` and `pf-icon--lg` and `pf-icon-sm`. then we'll make it even more modular.